app/javascript/shared/utils/JsTus.re

```diff
@@ -1,20 +1,19 @@
 [@bs.deriving abstract]
 type uploadOptions = {
   uploadUrl: string,
```

Comment:
Assuming we're looking at [the same library](https://github.com/tus/tus-js-client), should this argument be `endpoint`? 🤔

Reply:
Actually, I got that from [here](https://github.com/tus/tus-js-client/blob/master/docs/usage.md) check Upload to Vimeo.

Reply:
Oh, it's a Vimeo-specific thing. Good find.